src/contrib/cluster/Akka.Cluster.Sharding/ClusterSharding.cs

```diff
@@ -595,7 +784,7 @@ private void Done(bool ok)
     /// Check if we've received a shard start request.
     /// </summary>
     [Serializable]
-    public sealed class ResendShardHost
+    internal sealed class ResendShardHost
```

Comment:
IMO, it is somewhat arcane to have a serializable type where the type is not visible to the serializer.
(Most serializers will just ignore this and do magic anyway, but still, somewhat blurred intentions)